lakefile.lean

```diff
@@ -18,12 +18,14 @@ package mathlib where
 ## Mathlib dependencies on upstream projects.
 -/
 
-require batteries from git "https://github.com/leanprover-community/batteries" @ "main"
-require Qq from git "https://github.com/leanprover-community/quote4" @ "master"
-require aesop from git "https://github.com/leanprover-community/aesop" @ "master"
-require proofwidgets from git "https://github.com/leanprover-community/ProofWidgets4" @ "v0.0.38"
-require Cli from git "https://github.com/leanprover/lean4-cli" @ "main"
-require importGraph from git "https://github.com/leanprover-community/import-graph.git" @ "main"
+meta if get_config? doc = some "on" then -- do not download and build doc-gen4 by default
+require «doc-gen4» from git "https://github.com/leanprover/doc-gen4" @ "main"
+
```

Comment:
```suggestion
```

Reply:
This was a bad merge. We just removed this on master.